backend/grants/models.py

```diff
@@ -189,6 +189,12 @@ class ApprovedType(models.TextChoices):
         _("applicant reply deadline"), null=True, blank=True
     )
     applicant_message = models.TextField(_("applicant message"), null=True, blank=True)
+    plain_thread_id = models.CharField(
+        _("Plain threadID"),
+        max_length=50,
```

Comment:
Why the max length? Is this 50 correct?

Reply:
I put a bit longer. think it's 29 really